packages/manager/apps/web/client/app/dns-zone/history/history.routing.js

```diff
@@ -0,0 +1,41 @@
+import controllerDiff from './diff/history.controller';
+import templateDiff from './diff/history.html';
+import controllerDashboard from './dashboard/dashboard.controller';
+import templateDashboard from './dashboard/dashboard.html';
```

Comment:
```suggestion
import historyController from './dns-zone-history/history.controller';
import historyTemplate from './dns-zone-history/history.html';
import dashboardController from './dashboard/dashboard.controller';
import dashboardTemplate from './dashboard/dashboard.html';
```

Reply:
is diff-tool-viewer ok for you since it is more a tool for viewing the diff (between dns zones)?

Reply:
![image](https://github.com/ovh/manager/assets/8767496/a051f019-e6d0-49ff-a306-a6f3d8fac0ee)

Reply:
Seen with sachin, we have chosen instead dns-history-diff-tool-viewer as a more comprehensive name